dht-node/src/dht_node/index.ts

```diff
@@ -30,8 +41,12 @@ export const startDHT = async (topic: string): Promise<void> => {
     const keyPair = DHT.keyPair(getSeed())
     logger.info(`keyPairDHT: publicKey=${keyPair.publicKey.toString('hex')}`)
     logger.debug(`keyPairDHT: secretKey=${keyPair.secretKey.toString('hex')}`)
+    // insert or update keyPair in .env.devop file
+    setDevOpEnvValue('HOME_COMMUNITY_PUBLICKEY', keyPair.publicKey.toString('hex'))
+    setDevOpEnvValue('HOME_COMMUNITY_PRIVATEKEY', keyPair.secretKey.toString('hex'))
```

Comment:
this defeats the purpose of ever changing seeds by not setting it - we should discuss if we want this - currenlty our decision was `Production/Stage systems have stable keys, development systems generate new keys on every run`. Not beeing able to share the generated secret would limit our poibilities and is one open question in https://github.com/gradido/gradido/pull/2970

Reply:
removed